app/services/email_sender/notify_sender.rb

```diff
@@ -0,0 +1,34 @@
+require "notifications/client"
+
+module EmailSender
+  class NotifySender
+    def call(address:, **keyword_args)
+      send(address: address, **keyword_args)
+    end
+
+  private
+
+    def send(address:, **keyword_args)
+      client.send_email(
+        email_address: address,
+        template_id: template_id(keyword_args)
+      )
+    end
```

Comment:
I wonder if we should call this something other than `#send` to avoid clashing with [`Object#send`](https://ruby-doc.org/core-2.4.2/Object.html#method-i-send)? Not that it really matters.

Reply:
good point thanks @tuzz fixed that in 49de7db462496913b6feced6d7f84f5c64e81a4d